kpc/models.py

```diff
@@ -60,13 +60,17 @@ class Certificate(models.Model):
     HS_CODE_CHOICES = (
         ('7102.10', '7102.10'),
         ('7102.21', '7102.21'),
-        ('7102.29', '7102.29'),
         ('7102.31', '7102.31'),
-        ('7102.39', '7102.39'),
     )
 
     VOID_REASONS = ['Printing Error', 'Typographical error', 'No longer needed', 'Other']
 
+    # Price in USD of a certificate
+    PRICE = 20
+
+    # Days from issue upon which a certificate expires
+    EXPIRY_DAYS = 60
+
```

Comment:
Are we sure we want to hard-code these values? If there's a possibility of them changing, even if it's only every few years or by law, we might want to stick them in their own table with date ranges attached.

Reply:
Good point @annalee. I think this, along with the other Certificate specifications here, are great candidates to be moved into their own model which is editable via the admin. I'll open another issue for discussion.